packages/oasgraph/test/government_social_work_api.test.js

```diff
@@ -75,10 +75,3 @@ test('Get resource', () => {
   let errors = validate(createdSchema, ast)
   expect(errors).toEqual([])
 })
-
-test('Strict mode throws exception', () => {
```

Comment:
This test does not do anything... LinkNameCollision error was not used so an error was never thrown. I think we will need to add a new test for strict mode.

Reply:
All right.